Kickstarter-iOS/Features/BetaTools/Views/BetaToolsFooterView.swift

```diff
@@ -33,6 +33,7 @@ final class BetaToolsFooterView: UIView {
     )
   }
 
+  @available(*, unavailable)
```

Comment:
I did not, but [I looked it up!](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/attributes/#available)

This is saying "for all platforms, this method is unavailable." So if you try to call it, you get a compiler error:
<img width="722" alt="Screenshot 2024-04-04 at 9 13 41 AM" src="https://github.com/kickstarter/ios-oss/assets/146007185/32642341-ddb3-4ff9-b2b9-2ca17c48caaf">

This feels safe to me, since it's simply the compiler warning us about something that already would have crashed anyways.